presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2883,6 +2884,61 @@ public void testGroupedExecution()
         };
     }
 
+    @Test
+    public void testWriteStagingFiles()
```

Comment:
nit: `testWriteWithStagineFileEnabled` ?

Reply:
Named it `testInsertAndCreateTableWithWritingStagingFileEnabled`.